firewall/init.sls

```diff
@@ -6,18 +6,9 @@ iptables:
 
 {%- if firewall.enabled %}
 
-/etc/iptables-rules:
-  file.managed:
-    - source: salt://firewall/templates/iptables-rules
-    - template: jinja
-    - user: root
-    - group: root
-
-
-firewall-apply:
+firewall-save:
   cmd.run:
-    - name: 'cat /etc/iptables-rules | /sbin/iptables-restore'
-    - unless: diff -u -B <( iptables-save | sed -r 's/\[[0-9]+\:[0-9]+\]/[0:0]/' | grep -v '^#') <( cat /etc/iptables-rules )
+    - name: 'iptables-save > /etc/iptables.rules'
```

Comment:
You aren't calling this anywhere currently - should it be a notify(or watch?) from the `cmd: firewall-file-{{service}}-{{proto}}-{{port}}`?